app/src/main/java/com/example/pinterbest/utilities/Validator.kt

```diff
@@ -0,0 +1,179 @@
+package com.example.pinterbest.utilities
+
+import android.util.Patterns
+import android.widget.EditText
+import com.google.android.material.textfield.TextInputLayout
+import java.util.regex.Pattern
+
+class Validator private constructor() {
+    companion object {
+
+        // Default validation messages
+        private val PASSWORD_POLICY =
+            """Password should be minimum 8 characters long,
+            |should contain at least one capital letter,
+            |at least one small letter,
+            |at least one number and
+            |at least one special character among ~!@#$%^&*()-_=+|[]{};:'\",<.>/?""".trimMargin()
+
+        private const val NAME_VALIDATION_MSG = "Enter a valid name"
+        private const val EMAIL_VALIDATION_MSG = "Enter a valid email address"
+        private const val PHONE_VALIDATION_MSG = "Enter a valid phone number"
```

Comment:
This and others texts which are showed to user must be placed in string resources files

Reply:
Fixed here: https://github.com/enginire4ek/Pinterbest/pull/1/commits/c349b66d96a4b6bbf03557b4bc3771de1638810b